specifyweb/frontend/js_src/lib/components/Merging/definitions.ts

```diff
@@ -18,6 +19,7 @@ export const recordMergingTableSpec: Partial<{
       return hasGroup ? resource : undefined;
     },
     dialogHeader: mergingText.agentMergingWarning(),
-    dialogText: mergingText.agentContainsGroupDescription(),
+    dialogText: mergingText.cannotBeMerged(),
```

Comment:
(optional)

my point about making `dialogText` be generic was that you don't have to specify it here for every table - instead, you would just hardcode it in the `<InvalidMergeRecordsDialog>` since it would be the same for all tables

thus, you could remove it from `dialogText`
once that is removed, you can rename `dialogSpecificText` back to `dialogText`

sorry if this isn't clear

Reply:
Got it for the table name. Although we created two separate strings to have them displayed as 2 lines in the dialog with different styling. 
My suggestion is to have tableMergingWarning instead of agentMergingWarning and to be able to specify the table name in the string.